### PR TITLE
feat(react): add template

### PR DESCRIPTION
**Pull request by Fernans96**

### CODE REVIEW

File: react-project/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "dependencies": {
+    "css-loader": "^0.28.1",
+    "file-loader": "^0.11.1",
+    "node-sass": "^4.5.2",
+    "react": "^15.5.4",
+    "react-dom": "^15.5.4",
+    "sass-loader": "^6.0.3",
+    "style-loader": "^0.17.0",
+    "webpack": "^2.4.1"
+  }
```
Comment (Sy1v4in) [inline react-project/package.json]: Don't use `^` notation, use exact dependencies or `~` notation

File: techio.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+#  - title: Another Page
+#    statement: markdowns/another-page.md
+
+
+# Settings for your programming projects and the associated Docker images
+projects:
+  angular:
+    root: /react-project # Set the root path to your angular project
+    # Set the docker image. This image runs npm install for you on every build
+    runner: techio/node-npm-runner:1.1.0-node-7.4
```
Comment (Sy1v4in) [inline techio.yml]: Can't we use the official docker node image with a `buildCommand`?

File: markdowns/welcome.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Welcome!
+
+This React template lets you get started quickly with a simple working example using React (with `npm`). If it is your first contribution then you should have a look at the [Getting Started](https://tech.io/doc/getting-started-create-playground) document.
```
Comment (Sy1v4in) [inline markdowns/welcome.md]: npm or yarn?

File: techio.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+plan:
+  - title: Welcome # Set the page title (only displayed if your contribution has several pages)
+    statement: markdowns/welcome.md # Set the file path to the page content
+#  - title: Another Page
+#    statement: markdowns/another-page.md
+
+
+# Settings for your programming projects and the associated Docker images
+projects:
+  angular:
```
Comment (Sy1v4in) [inline techio.yml]: react

File: techio.yml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+    root: /react-project # Set the root path to your angular project
```
Comment (Sy1v4in) [inline techio.yml]: react

File: react-project/package.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+  },
+  "dependencies": {
+    "css-loader": "^0.28.1",
+    "file-loader": "^0.11.1",
+    "node-sass": "^4.5.2",
+    "react": "^15.5.4",
+    "react-dom": "^15.5.4",
+    "sass-loader": "^6.0.3",
+    "style-loader": "^0.17.0",
+    "webpack": "^2.4.1"
```
Comment (Sy1v4in) [inline react-project/package.json]: All the `sass` stuff is really long to install increasing the build time. Don't think it is mandatory for the template. We can use `css` directly don't we?

File: markdowns/welcome.md (hunk 3 above)
Comment (Fernans96) [reply markdowns/welcome.md]: npm

File: react-project/package.json (hunk 6 above)
Comment (Sy1v4in) [inline react-project/package.json]: `css-loader`, `file-loader`, `style-loader` and `webpack` should be dev dependencies.

File: markdowns/welcome.md (hunk 3 above)
Comment (Sy1v4in) [reply markdowns/welcome.md]: But with the last docker node image you can use `yarn` which is a bit faster than npm…

File: techio.yml (hunk 5 above)
Comment (Fernans96) [reply techio.yml]: 009f6e761ea8d7d610cd0b903cdcc74ebdfd53f2

File: techio.yml (hunk 4 above)
Comment (Fernans96) [reply techio.yml]: 009f6e761ea8d7d610cd0b903cdcc74ebdfd53f2

File: react-project/package.json (hunk 1 above)
Comment (Fernans96) [reply react-project/package.json]: 3c23ba20b17243a78e750f9ba3adfb4bd20b91ad

File: react-project/package.json (hunk 6 above)
Comment (Fernans96) [reply react-project/package.json]: 3c23ba20b17243a78e750f9ba3adfb4bd20b91ad

Comment (Fernans96) [reply react-project/package.json]: 3c23ba20b17243a78e750f9ba3adfb4bd20b91ad

File: markdowns/welcome.md (hunk 3 above)
Comment (Fernans96) [reply markdowns/welcome.md]: 60aba8492b75f78e969c436bee0ab6b1070f8340

**Review by Sy1v4in — APPROVED** [review]

**Pull request merged**